telegram/constants.py

```diff
@@ -824,6 +826,63 @@ class ChatLimit(IntEnum):
     """
 
 
+class BackgroundTypeLimit(IntEnum):
+    """This enum contains limitations for :class:`telegram.BackgroundTypeFill`,
+    :class:`telegram.BackgroundTypeWallpaper` and :class:`telegram.BackgroundTypePattern`.
+    The enum members of this enumeration are instances of :class:`int` and can be treated as such.
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    MIN_DIMMING = 0
```

Comment:
just double checking: I vaguely recall a discussion where we decided for which cases we need both MIN and MAX and for which cases MAX suffices. I have a hunch that we skipped min if it's 0 - could you kindly double check for consistency?

Reply:
Can confirm. All MIN_* values are >= 1.
Adjusted accordingly.